android/src/main/java/com/brentvatne/react/ReactVideoPackage.java

```diff
@@ -9,6 +9,7 @@
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.uimanager.ViewManager;
 
+import java.util.ArrayList;
```

Comment:
no need of this import

Reply:
Why are using it at line 29 (ReactVideoPackage)

```java
 List<NativeModule> modules = new ArrayList<NativeModule>();
 ```